### PR TITLE
Build app in workflow and add to container

### PR DESCRIPTION
**Pull request by sanjP10**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,15 +4,13 @@ ENV NODE_VERSION 16.8.0
...
-RUN yarn 
-RUN yarn global add cross-env
-RUN yarn build
+ADD ./node_modules ./node_modules
+ADD ./.next ./.next
 
 EXPOSE 80
 
-CMD [ "cross-env", "NODE_ENV=production", "yarn", "start"]
+CMD [ "npx", "next", "start"]
```
Comment (philipbeel) [inline Dockerfile]: Any reason to avoid cross-env?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -62,6 +65,8 @@ jobs:
         with:
           username: limitless86
           password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: LIST
```
Comment (philipbeel) [inline .github/workflows/main.yml]: What's this for?

**Review by philipbeel — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (sanjP10) [reply Dockerfile]: cross-env is mainly for windows, no need to bloat your container with cross-env, if NODE_ENV is perfectly find

File: .github/workflows/main.yml (hunk 2 above)
Comment (sanjP10) [reply .github/workflows/main.yml]: just debugging something atm

**Pull request merged**